### DISCUSSION TITLE
Domain Model Architecture Diagram outdated?

### DISCUSSION THREAD

**Discussion by ghost** [eclipse-tractusx/tractusx-edc#1015] [General]
There's a domain model architecture diagram here in the repo:
https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/usage/README.md

According to feedback from core-edc, it is no longer up to date:
https://github.com/eclipse-edc/Connector/discussions/3815#discussioncomment-8334573

Could you please update the architecture diagram if it needs to be updated? It led to some confusion.

**Comment by ndr-brt** [comment]
my suggestion would be to delete it, documentation about the domain model is contained in the [EDC developer handbook](https://eclipse-edc.github.io/docs/#/documentation/developer/handbook)

**Reply by SebastianOpriel** [reply]
but unfortunately there is no graphic, or do I miss something?

Onother one I found [here](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/99b607f615c8db42cbcffe05a7eef2ef2740bafe/static/img/domain_model.drawio.svg): 
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/22075788/9674d77d-696f-4316-9426-ffc25dc8d79f)
But indeed, simply the same image as referenced by @tmberthold but in another style (drawio)

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue

**Reply by SebastianOpriel** [reply]
This is indeed still a problem...

**Reply by ndr-brt** [reply]
let's remove it altogether: https://github.com/eclipse-tractusx/tractusx-edc/issues/1819

**Comment by lgblaumeiser** [comment]
Close this, as #1819 is already worked on
